Description: Correct english spelling errors
 Correct english spelling errors in the fusiondirectory-setup manpage.
Author: Benoit Mortier <benoit.mortier@opensides.be>
Last-Update: 2012-06-06
Index: pkg/contrib/man/fusiondirectory-setup.1
===================================================================
--- pkg.orig/contrib/man/fusiondirectory-setup.1  2012-06-06 17:34:49.000000000 +0200
+++ pkg/contrib/man/fusiondirectory-setup.1 2012-06-06 17:35:09.000000000 +0200
@@ -167,10 +167,10 @@
 This option will list the variables you can change to install FusionDirectory on another set of directories. This option is intended for people wanting to install from the sources.
 .IP "\-\-set\-VAR=variable" 4
 .IX Item "--set-VAR=variable"
-This option will change the variable for the FusionDirectory installation. it is only usefull with \-\-install\-directories and for people installing from sources.
+This option will change the variable for the FusionDirectory installation. it is only useful with \-\-install\-directories and for people installing from sources.
 .IP "\-\-write\-vars" 4
 .IX Item "--write-vars"
-This option will write back the variables.inc file with the updated variables and is only usefull with \-\-set\-VAR=variable and for people installing from sources.
+This option will write back the variables.inc file with the updated variables and is only useful with \-\-set\-VAR=variable and for people installing from sources.
 .IP "\-\-yes" 4
 .IX Item "--yes"
 This flag will answer \*(L"yes\*(R" to every yes/no question asked by the script
